README.md

```diff
@@ -25,3 +26,13 @@ Change:
               MYSQL_PASSWORD - MySQL password
               MQTT_HOST - MQTT hostname or IP (If MQTT_HOST is not specified, MQTT support will not be enabled)
               /tmp - preferred location on the host
+              
+              User / Group Identifiers
```

Comment:
Not sure if it's required for this image...
Update, removing preceding whitespace...